src/main/java/com/comphenix/protocol/events/ListenerOptions.java

```diff
@@ -2,29 +2,24 @@
 
 import com.comphenix.protocol.injector.GamePhase;
 
-
 /**
  * Represents additional options a listener may require.
- * 
+ *
  * @author Kristian
  */
 public enum ListenerOptions {
+
 	/**
-	 * Retrieve the serialized client packet as it appears on the network stream.
-	 */
-	INTERCEPT_INPUT_BUFFER,
```

Comment:
Why did you remove this enum? What to use instead?

What about the error com.comphenix.protocol.events.PacketEvent.getNetworkMarker() throwing exception, which asks to "Add the option ListenerOptions.INTERCEPT_INPUT_BUFFER to your listener."
@derklaro 
@dmulloy2

Reply:
There is no replacement for this. See my initial PR comment for details